Dockerfile

```diff
@@ -1,14 +1,14 @@
-# Needed for DynamoDB Local, which is used for unit tests
-FROM openjdk:15-alpine3.11
+# Java JDK is needed for DynamoDB Local, which is used for unit tests
```

Comment:
To get AWS SDK v3 support for the local DynamoDB, we have to upgrade `jest-dynamodb`, which necessitates upgrading to a newer version of Nodejs, which necessitates switching from the deprecated base Docker image we were using to this one.